### ISSUE TITLE
Bounding box wrap line gdImageStringFT

### ISSUE THREAD

**Issue by ReznicencuBogdan** [libgd/libgd#829]
Hi! The documentation is not that clear on the behaviour of `gdImageStringFT`. 

###### Consider the following scenario: 

I need to print a long string onto some image. If I try calculate the bounding box of the string it exceeds the width of the image by far. Having newlines inside the string does help, for that breaks the string on multiple lines in the image. What I would like to know is if there is a setting to force the wrapping of text if it exceeds the width of the image. I know of the existence of `gdImageSetClip` however this function only restricts the drawing of out of bounds components.

**Comment by cmb69** [comment]
> What I would like to know is if there is a setting to force the wrapping of text if it exceeds the width of the image.

No, that is not (yet) supported. Instead you'd need to do this yourself. Calling `gdImageStringFT()` with a `NULL` image is supported (but not documented), and that allows to retrieve the bounding rectangle without actually rendering the glyphs. Then break (and align) the text as desired.

**Comment by ReznicencuBogdan** [comment]
Thanks for the response. I ended up doing just that - i.e calling gdImageStringFT() with a NULL image multiple times thus determining at each step the best position to break the text. Guess this should belong to the feature request list.
